table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueDatabase.java

```diff
@@ -386,6 +386,11 @@ public UdbTable getTable(String tableName) throws IOException {
               .withTableName(tableName);
       if (glueClient.getPartitions(getPartitionsRequest).getPartitions() != null) {
         partitions = glueClient.getPartitions(getPartitionsRequest).getPartitions();
+        LOG.debug("Get {} partitions from Table {} Database {}.",
+            partitions.size(), tableName, mGlueDbName);
+        partitions.stream().forEach(partition ->
```

Comment:
This should be guarded by `LOG.isDebugEnabled()`, since `toString` could be expensive.

Also, the message should be distinct and descriptive, so it could be something like "Glue table {}.{} with partition: {}"

Reply:
Changed.